cloudca/README.md

```diff
@@ -262,8 +264,9 @@ resource "cloudca_volume" "data_volume" {
 - service_code - (Required)
 - environment_name - (Required)
 - name - (Required) The name of the volume to be created
-- storage_tier - (Required) Either performance, intermediate or standard
-- size_in_gb - (Required) The size of the volume. Must be a valid size for the chosen storage tier
+- disk_offering - (Required) The name or id of the disk offering to use for the volume
+- size_in_gb - (Optional) The size of the volume. Only for disk offerings with custom size.
```

Comment:
Maybe add size of the volume "in GB", because of the lower case "gb" in the field name